sdks/java/core/src/test/java/org/apache/beam/sdk/util/common/ReflectHelpersTest.java

```diff
@@ -139,12 +139,12 @@ public void testTypeFormatterWithGenerics() throws Exception {
   public void testToStringForPrint() throws Exception {
```

Comment:
testToStringForPrint -> testAnnotationFormatter

Add a test for a default value with a . and ( and $ in it so that we remember why we have string parsing logic setup the way we do.